fastlane/actions/verify_keys.rb

```diff
@@ -0,0 +1,42 @@
+module Fastlane
+  module Actions
+
+    class VerifyKeysAction < Action
+      def self.run(params)
+
+        # Ensure no deploys with OSS keys
+        require 'cocoapods-core'
+        puts "Validating CocoaPods Keys"
+
+        podfile = Pod::Podfile.from_file "Podfile"
```

Comment:
This integration looks really cool - it looks like it's useful for everyone using CocoaPods Keys? Would you want to add this to the built-in fastlane actions?